components/core/src/clp/regex_utils/constants.hpp

```diff
@@ -14,6 +36,13 @@ constexpr char cRegexStartAnchor{'^'};
 constexpr char cRegexEndAnchor{'$'};
 constexpr char cEscapeChar{'\\'};
 constexpr char cCharsetNegate{'^'};
+
+// Character bitmaps
+// This is a more complete set of meta characters than necessary, as the user might not be fully
+// knowledgeable on which meta characters to escape, and may introduce unnecessary escape sequences.
+constexpr auto cRegexEscapeSeqMetaChars = create_char_bit_array("*+?|^$.{}[]()<>-_/=!\\");
+// This is the set of meta characters that need to be escaped in the wildcard syntax.
+constexpr auto cWildcardMetaChars = create_char_bit_array("?*\\");
```

Comment:
Do you think it's more clear to use explicit names about look up table, such as `cRegexEscapeSeqMetaCharLUT`?